src/main/java/com/couchbase/lite/support/Batcher.java

```diff
@@ -161,6 +162,12 @@ public void queueObjects(List<T> objects) {
         }
     }
 
+    public void flushAllAndWait() {
+        synchronized (flushAllMutext) {
+            flushAll(true);
+        }
+    }
+
```

Comment:
Today, 02/08/17, is just a day before FC (Feature Complete). We will work for this during next release cycle.